src/cli/build.ts

```diff
@@ -29,12 +30,26 @@ export default function (program: RootCmd) {
       "-r, --registry-info [<registry>/<username>]",
       "Registry Info: Image registry and username. Note: You must be signed into the registry",
     )
+    .option("-o, --output-dir [output directory]", "Define where to place build output")
     .addOption(
       new Option("--rbac-mode [admin|scoped]", "Rbac Mode: admin, scoped (default: admin)")
         .choices(["admin", "scoped"])
         .default("admin"),
     )
     .action(async opts => {
+      // assign custom output directory if provided
+      if (opts.outputDir) {
+        outputDir = opts.outputDir;
+        createDirectoryIfNotExists(outputDir).catch(error => {
+          console.error(
+            "Could not create output directory. ",
+            error,
+            " Defaulting to dist folder.",
+          );
+          outputDir = "dist";
+        });
```

Comment:
I think... I like the error message, but I'm not sure I like the use-dist-by-default, personally. :thinking: 

It feels a bit like it violates the [Principle of Least Surprise](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) in that, if a user is telling Pepr to _not_ use the default directory (ostensibly, for a reason), then Pepr probably _shouldn't_ use it.

Personally, if I'm saying to Pepr, "Do your thing, Pepr, but put the results 'here'" and "here" isn't available (because it's unreachable / or I made a typo or whatever) then I'd want Pepr to just tell me that & exit.  Cuz that's the quickest way for me correct my mistake & try again.

If we just push on using defaults then we're essentially telling the caller that:
1. we know what they "really" want (which isn't what they asked for),
2. it might take them a while for a user to _notice_ that what's happening isn't what they asked for (cuz lots of people don't watch CLI output all that closely as it scrolls by), and
3. it introduces the possibility that we're stomping on things that the user might really want to keep (since, by passing the -o flag, they were attempting to instruct Pepr _not_ to touch the default location).

Reply:
You are right, lets stick to best practices when it comes to user interface. Changed